target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -152,6 +152,10 @@ dir-860l-b1)
 edimax,br-6478ac-v2)
 	set_wifi_led "$boardname:blue:wlan"
 	;;
+esr600)
```

Comment:
Needs to be "engenius,esr600". The others, too.

Reply:
Oops. Fixing it there and in the 02_network cases.